omegaconf/omegaconf.py

```diff
@@ -727,7 +727,12 @@ def select(
 
     @staticmethod
     def update(
-        cfg: Container, key: str, value: Any = None, merge: Optional[bool] = None
+        cfg: Container,
+        key: str,
+        value: Any = None,
+        *,
+        merge: Optional[bool] = None,
```

Comment:
Probably not a big deal, but just flagging that making `merge` keyword-only is an undocumented breaking change

Reply:
yeah, I am okay with this one.
I am planning to change the default of merge to be True in the future and all the examples as well as the [deprecation warning link](https://github.com/omry/omegaconf/issues/367) shows it used with an a keyword.

Reply:
That future is actually now. Added #667.